src/SAWScript/Crucible/JVM/BuiltinsJVM.hs

```diff
@@ -6,6 +6,8 @@ Stability   : provisional
 -}
 
 
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE NoMonoLocalBinds #-}
```

Comment:
As noted in the commit message, I opted to just disable `MonoLocalBinds` rather than do the slightly more invasive fix of giving local definitions top-level type signatures. Let me know if you'd prefer the latter instead.